docs/api.md

```diff
@@ -304,6 +304,8 @@ for `code` and `errno` are:
   Unknown client_id
 * `code: 400, errno: 164`:
   Stale auth timestamp
+* `code: 409, errno: 165`:
```

Comment:
If this can be surfaced to the user, let's add a corresponding entry in the [content-server's auth-errors.js list](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/lib/auth-errors.js).

Reply:
Oh yeah, forgot about that!